nucleic-acids-research.csl

```diff
@@ -42,15 +45,15 @@
     <text variable="title"/>
   </macro>
   <macro name="publisher">
-    <group delimiter=", ">
-      <text variable="publisher"/>
-      <text variable="publisher-place"/>
-    </group>
+    <text variable="publisher"/>
+    <text variable="publisher-place"/>
   </macro>
```

Comment:
@POBrien333, would you mind taking a look at this style? This change in particular can't be right (no delimiter). See https://academic.oup.com/nar/pages/Ms_Prep_Submission#preparation for the current documentation link.

Reply:
Correct. 
This should be:
`  <macro name="publisher">
    <group delimiter=", ">
      <text variable="publisher"/>
      <text variable="publisher-place"/>
    </group>
  </macro>`